api/envoy/config/route/v3/route_components.proto

```diff
@@ -1929,6 +1929,10 @@ message HeaderMatcher {
   // * The regex ``\d{3}`` does not match the value *1234*, so it will match when inverted.
   // * The range [-10,0) will match the value -1, so it will not match when inverted.
   bool invert_match = 8;
+
+  // If true, indicates the exact_match/prefix_match/suffix_match/contains_match should be case
```

Comment:
Nit `*exact_match*`, `*safe_regex*` etc.

Reply:
updated to use `StringMatcher string_match` instead.